project-resources/INFRASTRUCTURE_MENU.md

```diff
@@ -3,12 +3,15 @@
 The OpenJS Foundation provides a number of services to support critical infrastructure for hosted projects.  We expect projects to be respectful of these services, to abide by their terms of use, and to be put into use for the good of the project and the OpenJS Foundation.
 
 ## Billing for services and mitigating the bus factor
-**For all project services, please add an OpenJS Foundation account at an owner or highest-level of permission access.**  This helps ensure continuity by reducing the bus factor on the project, and ensures you are never locked out.  It is also **required** in order for the OpenJS Foundation to pay service fees on behalf of your project.  Access to the OpenJS Foundation administrator/owner account will never be shared with others, and will only be granted to operations, IT, and finance staff at the Linux Foundation.
+**For all project services, please add an OpenJS or Linux Foundation account at an owner or highest-level of permission access.**  This helps ensure continuity by reducing the bus factor on the project, and ensures you are never locked out.  It is also **required** in order for the OpenJS Foundation to pay service fees on behalf of your project.  Access to the OpenJS Foundation administrator/owner account will never be shared with others, and will only be granted to operations, IT, and finance staff at the Linux Foundation.
 
 If you don’t know the name of the OpenJS Foundation account for a service, please contact operations@openjsf.org.
 
 ## Websites
-Digital Ocean offers free droplets to OpenJS Foundation projects.  Projects are solely responsible for the content and design of their websites.
+Digital Ocean offers free droplets to OpenJS Foundation projects, or projects can use GitHub Pages for free.  Projects are solely responsible for the content and design of their websites.
+
+Resources and base themes (please contribute other templates as you find them):
+* The [Amethyst theme](https://github.com/qunitjs/jekyll-theme-amethyst) is maintained by @krinkle for use with GitHub Pages.
```

Comment:
Extremely -1 on us calling out free things in this document. It devalues the non-free things that the Foundation *does* provide.